python/rikai/spark/sql/codegen/mlflow_registry.py

```diff
@@ -215,6 +215,8 @@ def resolve(self, raw_spec):
         uri = raw_spec.getUri()
         logger.info(f"Resolving model {name} from {uri}")
         parsed = urlparse(uri)
+        if parsed.netloc:
```

Comment:
Could you add some document here, i.e. including the model url spec in mlflow?

Wanted to understand that `mlflow` does not actually support netloc field?

Reply:
The reason why `mlflow://` does not work is that our implementation does not allow `mlflow://`

I think it could be regarded as bug. And let fix it.